src/trainers/trainer-list.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+// import trainerService from "./trainer-service"
+import Button from '@mui/material/Button';
+import { DataGrid } from '@mui/x-data-grid';
+// const { useHistory } = window.ReactRouterDOM;
+// const { useState, useEffect } = React;
+
+const TrainersList = () => {
+    // const history = useHistory();
+    // const [trainers, setTrainers] = useState([]);
+    // useEffect(() => {
+    //   findAllUsers()
+    // }, [])
+    // const findAllUsers = () =>
+    //     trainerService.findAllUsers()
+    //     .then(trainers => setTrainers(trainers))
```

Comment:
AHHH THE COMMENTS
also everything looks good but I'd change the file name to be the same as the function name - i.e. `trainers-list.js` or `trainersList.js` whichever you prefer